development-strategy.md

```diff
@@ -34,8 +34,12 @@ __FRONTEND__
 * Home page needs
     * Responsive website
     __Header__
-    * Collapse navigation bar (`links` to other pages or one page, `sign up`, `sign in buttons`,`tutorials`,`practice-test`,`browse-tests`, `profile`)
+    * Collapse navigation bar (`links` to other pages or one page, `sign up`, `sign in buttons`,`tutorials`,`practice-test`,`browse-tests`, `profile`, `free sample test`, `our logo` )
     __Main Section__
+    * some fotos for advertisements
```

Comment:
Is it possible to upgrade all like these lines as example below:
* Create 3-4 `div` `class= ads` this section will include some fotos for advertisements